packages/chat-client/src/controllers/engine.ts

```diff
@@ -778,27 +773,25 @@ export class ChatEngine extends IChatEngine {
       // Subscribe to the chat thread topic.
       await this.client.core.relayer.subscribe(chatThreadTopic);
 
-      const { selfAccount, peerAccount } =
-        this.client.chatThreadsPending.get(topic);
+      const { inviteeAccount, inviterAccount } =
+        this.client.chatSentInvites.get(topic);
 
       await this.client.chatThreads.set(chatThreadTopic, {
         topic: chatThreadTopic,
-        selfAccount,
-        peerAccount,
+        selfAccount: inviterAccount,
+        peerAccount: inviteeAccount,
       });
 
       console.log("onInviteResponse > chatThreads.set: ", chatThreadTopic, {
         topic: chatThreadTopic,
-        selfAccount,
-        peerAccount,
+        selfAccount: inviterAccount,
+        peerAccount: inviteeAccount,
       });
 
-      // TODO (post-mvp): decide on a code to use for this.
-      await this.client.chatThreadsPending.delete(topic, {
-        code: -1,
-        message: "Peer accepted invite.",
+      //TODO: Delete after 3
```

Comment:
3? 😄

Reply:
Ah, did not complete the sentence. I'll update the comment. Should be `Delete after 3 settled invites`